src/frameworks/drupal7/solr.md

```diff
@@ -2,29 +2,48 @@
 
 ## Requirements
 
-You will need to add the [Search
-API](https://www.drupal.org/project/search_api) and [Search API
-Solr](https://www.drupal.org/project/search_api_solr) modules to your
-project.
+You will need to add the [Search API](https://www.drupal.org/project/search_api) and [Search API
+Solr](https://www.drupal.org/project/search_api_solr) modules to your project. The [Search API Override](https://www.drupal.org/project/search_api_override) module is strongly recommended in order to allow the Solr configuration to be populated from `settings.php`.
 
 If you are using a make file, you can add those lines to your
 `project.make`:
 
 ```ini
-projects[search_api][version] = 1.13
-projects[search_api_solr][version] = 1.x-dev
+projects[entity][version] = 1.8
+projects[search_api][version] = 1.20
+projects[search_api_solr][version] = 1.11
+projects[search_api_override][version] = 1.0-rc1
 ```
 
 ## Configuration
 
-Go to `/admin/config/search/search_api` and click on *Add Server*. Give
-the server a name and a description.
+The Search API module includes recommended configuration files to use with Drupal.  See the [Solr](/configuration/services/solr.md) configuration page for details of how to configure your Solr server to use the Drupal configuration files.  Note that the module does not include Drupal configuration files for Solr 6.  The Drupal 8 version of the module does, however, and should work acceptably.  It can also be customized as desired.
```

Comment:
"Note that the module does not include Drupal configuration files..."

Should this refer to the Drupal 7 module?